cluster-autoscaler/utils/nodegroupset/compare_nodegroups.go

```diff
@@ -101,6 +118,12 @@ func IsNodeInfoSimilar(n1, n2 *schedulercache.NodeInfo) bool {
 			if label == kubeletapis.LabelZoneRegion {
 				continue
 			}
+			if label == kubeletapis.LabelGardenerWorkerGroup {
```

Comment:
This is gardener specific change. Whereas the [above](https://github.com/gardener/autoscaler/pull/23/files#diff-0da093ed8e25f6d92d7d334106d395baR79) is an upstream change. Isn't better to keep them in separate commits?

Reply:
thanks for the review. done.